README.md

```diff
@@ -173,3 +178,24 @@ RxJavaPlugins.setErrorHandler { throwable ->
   }
 }
 ```
+
+#### Which permissions are needed?
+**Android**
+
+For android the library uses the following permissions:
+* ACCESS_FINE_LOCATION : this permission is needed because old Nexus devices need location services in order to provide reliable scan results
+* BLUETOOTH : allows apps to connect to a paired bluetooth device
+* BLUETOOTH_ADMIN: allows apps to discover and pair bluetooth devices
+
+These permissions are already added in the manifest of the this library and thus should automatically merge
+into the manifest of your app. It is not needed to add the permissions in your manifest.
+
+**iOS**
+
+For iOS it is required you add the following entries to the info.plist file of your app. It is not required to access Core BLuetooth without this. See [our example app](https://github.com/PhilipsHue/flutter_reactive_ble/blob/master/example/ios/Runner/Info.plist) on how to implement this. For more indepth details: [Blog post on iOS bluetooth permissions](https://medium.com/flawless-app-stories/handling-ios-13-bluetooth-permissions-26c6a8cbb816)
```

Comment:
Do you mean "allowed" instead of "required'?

Reply:
I guess, @PieterAelse means this sentence:

```markdown
It is not required to access Core BLuetooth without this.
```

Additionally, the "info.plist" should be "Info.plist" (capitalized), and I'd put a file name in double quotes.

Reply:
good catch I mean indeed it is not allowed :)